packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html

```diff
@@ -1,75 +1,47 @@
-<div
-    class="d-flex"
-    data-ng-if=":: $index % 2 === 0"
-    data-ng-repeat="servicePack in ::$ctrl.servicePacks track by servicePack.name"
->
-    <div
-        class="orderSelection__product-container"
-        data-ng-if=":: $ctrl.servicePacks[i]"
-        data-ng-repeat="i in ::[$index, $index + 1]"
+<div class="row mb-3">
+    <oui-select-picker
+        class="col-md-4 mb-3"
+        data-ng-repeat="servicePack in ::$ctrl.servicePacks track by servicePack.name"
+        name="servicePack"
```

Comment:
```suggestion
        data-name="servicePack"
```

Reply:
name is a standard HTML attribute. date- is not required.